wled00/usermods_list.cpp

```diff
@@ -9,7 +9,7 @@
  * || || ||
  * \/ \/ \/
  */
-//#include "../usermods/EXAMPLE_v2/usermod_v2_example.h"
```

Comment:
Why did you remove the example?

Reply:
Sorry, I was doing some tests on my own, I didn't know what it affected here